frontend/packages/shared/src/components/molecules/AltinnMenuItem.tsx

```diff
@@ -25,7 +25,7 @@ function AltinnMenuItem(props: IAltinnMenuItemProps, ref: React.Ref<HTMLLIElemen
       classes={{ root: classes.menu }}
     >
       <ListItemIcon classes={{ root: classes.icon }}>
-        <i className={iconClass} />
+        {IconComponent && <IconComponent />}    
```

Comment:
Unødvendige mellomrom
```suggestion
        {IconComponent && <IconComponent />}
```